bin/main.rb

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
+class Player
+  attr_accessor :name, :movement
+
+  def initialize(name, movement)
```

Comment:
_Kindly fix this_
bin/main.rb:5:in `initialize': wrong number of arguments (given 0, expected 2) (ArgumentError)